src/main/java/com/irum/orderservice/domain/deliveryaddress/repository/DeliveryAddressRepositoryImpl.java

```diff
@@ -12,12 +13,12 @@
 @Repository
 @RequiredArgsConstructor
 public class DeliveryAddressRepositoryImpl
-        implements com.irum.orderservice.domain.deliveryaddress.repository
+        implements com.irum.orderservice.domain.deliveryaddress.domain.repository
                 .DeliveryAddressRepositoryCustom {
     private final JPAQueryFactory queryFactory;
 
     private BooleanExpression belongsToMember(Long memberId, QDeliveryAddress deliveryAddress) {
-        return deliveryAddress.member.memberId.eq(memberId);
+        return deliveryAddress.member.eq(memberId);
```

Comment:
p4 : deliveryAddress 엔티티 필드 명을 member -> memberId로 수정하시면서 여기도 같이 수정해주세요! 
```suggestion
        return deliveryAddress.memberId.eq(memberId);
```